modules/go-health-monitoring.adoc

```diff
@@ -0,0 +1,19 @@
+[id="health-information-resources_{context}"]
+= Checking health information
+
+.Prerequisites
+
+* {gitops-title} Operator installed from *OperatorHub*.
+* Argo CD applications are in sync.
+
+.Procedure
+
+* The Operator will install the backend service in `openshift-gitops` namespace. Click *Environments* under the *Developer* perspective. 
+
+*  The *Environments* page shows the list of applications along with their *Environment status*.
+
+* Hover over the icons under the *Environment status* column to see the sync status of all the environments.
+
+* Click on the application to view the details of the application.
+
+* If the application is out of sync or not healthy the corresponding icon is displayed under *Resources*. Hover over the icons to see the health status and the sync status. 
```

Comment:
That should probably be a troubleshooting section. Tell the user what they can do in case the application is out of sync or not healthy.

Reply:
what does "the corresponding icon" mean? What does it look like, what is it named? It would be better to be specific here.

Reply:
Why is it one corresponding icon but the recommendation is to hover over icons in plural?

Reply:
not sure about the troubleshooting section, this article is on checking the health status

Reply:
In that case, ignore my comment :)